src/api/localStorage.ts

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+export function useLocalStorage(key: string, initialState: Todo[]) {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
export const useLocalStorage = (key: string, initialState: Todo[]) => {
```